modules/custom/az_migration/migrations/az_media.yml

```diff
@@ -52,7 +52,7 @@ process:
       method: process
       value: 'video/oembed'
     - plugin: default_value
-      source: '@mfid'
+      default_value: '@mfid'
```

Comment:
```suggestion
      default_value: '@mfid'
      source: '@mfid'
```

Reply:
You can see the changes I tried in my other PR below.
- https://github.com/az-digital/az_quickstart/pull/4435/commits/2a77af08134998d737b75aca0f2145afbb31a0f8
- https://github.com/az-digital/az_quickstart/pull/4435/commits/7541d3e86cfc47528972d30fdc85deb5280442cc
- https://github.com/az-digital/az_quickstart/pull/4435/commits/2d1bd4fc2d6a63de3e07dba0709d61254f8c22a6

Reply:
<img width="1222" height="476" alt="Image" src="https://github.com/user-attachments/assets/f09b5d9b-1ee4-46aa-b5da-698a03d31d0c" />